src/main/java/com/minecolonies/coremod/client/gui/WindowBarracksBuilding.java

```diff
@@ -0,0 +1,36 @@
+package com.minecolonies.coremod.client.gui;
+
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.colony.buildings.BuildingBarracks;
+import com.minecolonies.coremod.colony.buildings.BuildingWareHouse;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.colony.buildings.BuildingWareHouse'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)